greenplumpython/table.py

```diff
@@ -630,17 +633,19 @@ def _fetch(self, is_all: bool = True) -> Iterable[Tuple[Any]]:
         result = self._db.execute(to_json_table._build_full_query())
         return result if result is not None else []
 
-    def save_as(self, table_name: str, temp: bool = False, column_names: List[str] = []) -> "Table":
+    def save_into(
+        self, table_name: str, temp: bool = False, column_names: List[str] = []
+    ) -> "Table":
         """
-        Save the table to database as a real Greenplum Table
+        Save the **data** into a new :class:`Table`
 
         Args:
             table_name : str
```

Comment:
Need a short description for the `table_name` arg here.

Reply:
table_name: str: name of the table which can be later used with db.open_table(table_name) 
```suggestion
            table_name : str
```